package/INI/MPMaps.ini

```diff
@@ -8336,4 +8337,16 @@ Waypoint3=49036
 Waypoint4=50036
 Waypoint5=51036
 Waypoint6=49034
-Waypoint7=41049
\ No newline at end of file
+Waypoint7=41049
+
+[Maps\Yuri's Revenge\2_alpine_assault]
+Author=endless
+Description=[2] Alpine Assault
+EnforceMaxPlayers=True
+GameModes=Standard
```

Comment:
which gamemode should this map be in?

Reply:
Battle, I suppose? How should I modify that in the map itself? Or would I Modify `MPMaps.ini` directly?  

Wouldn't mind if it got added to the ladder as well although I'm not sure if that's a separate process.

Reply:
if you update the map file with Battle game mode, the tool will pick that up and apply it

for this map specifically go ahead and manually update mpmaps.ini, or update the map file and re-run the tool and confirm it gets updated

Reply:
Updated